tools/tools-common.c

```diff
@@ -235,23 +219,24 @@ tools_exec_command(const char *prefix, int real_argc, char **real_argv)
     }
 
     command = real_argv[0];
+#ifdef _MSC_VER
```

Comment:
Are you sure that's needed? Generally I think windows also accepts the unix separator.

Reply:
well, apparently the answer to that is "windows does accept it", or at least the CI passed with it dropped :)